app/models/host_alias.rb

```diff
@@ -0,0 +1,13 @@
+class HostAlias < ActiveRecord::Base
+  belongs_to :domain
```

Comment:
should it be unique in the context of a domain? e.g. you can't have more than one from a domain.